### PR TITLE
feat: visuals for add preference, preference option drawers

### PR DESCRIPTION
**Pull request by emilyjablonski**
# Pull Request Template

## Issue Overview

This PR addresses #2748 #2839

- [x] This change addresses only certain aspects of the issue

## Description

This PR targets partial AC from the above two issues and includes the visual pieces of the two new drawers for adding and editing preferences and preference options. 

The next PR will be functional, this is just the UI, to avoid a big PR.

## How Can This Be Tested/Reviewed?

Compare both drawers with those in the [Figma](https://www.figma.com/file/Y8bFd0aSz7eYJ2TCNid6s3/bloom-partners?node-id=5725%3A182890) in the issue.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [ ] I have exported any new pieces added to ui-components
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: backend/core/src/migration/1657568202436-turn-settings-off.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { MigrationInterface, QueryRunner } from "typeorm"
+import { CountyCode } from "../shared/types/county-code"
+
+export class turnSettingsOff1657568202436 implements MigrationInterface {
```
Comment (emilyjablonski) [inline backend/core/src/migration/1657568202436-turn-settings-off.ts:4]: At the moment this feature is toggled by a jurisdictional setting `enablePartnersSettings`. It was turned on for some jurisdictions when the migration was added. Now that we are able to have different seed data than migration data due to Colin's recent changes, we can turn the flag off here but turn it on in the seeds to continue to have it default be on locally.

File: ui-components/src/forms/Field.tsx [hunk 2]
```diff
@@ -74,7 +74,7 @@ const Field = (props: FieldProps) => {
     if (props.caps) labelClasses.push("field-label--caps")
     if (props.primary) labelClasses.push("text-primary")
     if (props.readerOnly) labelClasses.push("sr-only")
-    if (props.type === "checkbox" || props.type === "radio") {
```
Comment (emilyjablonski) [inline ui-components/src/forms/Field.tsx:77]: New in the designs, when a checkbox field has just a single value, it should not be bolded

**Comment by emilyjablonski** [comment]
Note: Due to new feedback, the `can you opt out` question is removed, and the opt out label will still be there

**Review by YazeedLoonat — APPROVED** [review]
LGTM, small change that may make things a lil smoother later

**Pull request merged**